src/main/java/org/terasology/gooeyDefence/EnemyManager.java

```diff
@@ -93,20 +149,15 @@ public void onPeriodicActionTriggered(PeriodicActionTriggeredEvent event, Entity
      * @param entranceNumber The entrance to spawn at
      */
     public void spawnEnemy(int entranceNumber) {
-
-        List<Vector3i> path = pathfindingSystem.getPath(entranceNumber);
-        if (path == null) {
+        if (!DefenceField.isFieldActivated()) {
```

Comment:
*comment unrelated to this line of code* in DefenceField.java line 38, the number 3 should be extracted into a constant.

Reply:
This has been fixed on master